MdePkg/Library/StackCheckLib/StackCheckLibCommon.c

```diff
@@ -0,0 +1,62 @@
+/** @file
+  Provides the required functionality for handling stack
+  cookie check failures.
+
+  Copyright (c) Microsoft Corporation.
+  SPDX-License-Identifier: BSD-2-Clause-Patent
+**/
+
+#include <Base.h>
+
+#include <Library/DebugLib.h>
+#include <Library/BaseLib.h>
+#include <Library/StackCheckFailureHookLib.h>
+
+/**
+  Calls an interrupt using the vector specified by PcdStackCookieExceptionVector

```

Comment:
'Calls' does not seem right term here.  Signal or trigger because the interrupt vector can not be directly called.

Reply:
I can update the language